untraced/yksg/src/lib.rs

```diff
@@ -277,15 +229,14 @@ impl StopgapInterpreter {
             .frames
             .first()
             .unwrap()
-            .mem
             .local_ptr(&yktrace::sir::INTERP_STEP_ARG);
         std::ptr::write::<*mut u8>(ptr as *mut *mut u8, ctx);
     }
 
     pub unsafe fn interpret(&mut self) -> bool {
         while let Some(frame) = self.frames.last() {
             let body = frame.body.clone();
-            let block = &body.blocks()[usize::try_from(frame.bbidx).unwrap()];
+            let block = &body.blocks()[usize::try_from(frame.pc).unwrap()];
```

Comment:
Make an accessor for `pc`?

Reply:
This is all in the same module, so we don't get much benefit from doing that.